### ISSUE TITLE
KB and Go should be translatable

### ISSUE THREAD

**Issue by scoid** [HUB-HUmanBeing/SystemD#247]
Sorry being a nudnik here, but again this should be translatable.

![Firefox_Screenshot_2020-01-24T07-18-14 981Z](https://user-images.githubusercontent.com/2099251/73051155-f7592180-3e79-11ea-80bc-9d62b3ed3b32.png)

**Comment by robinBanquo** [comment]
It is really nice that you are a nudnik, it is with your eagle eyes that we will produce a "near perfect" app :)
this will be done someday, there is more prioritary debugs, but when there is an issue, it is not lost.
if you want, I can show you how to correct it and all of these kind of i18n issues, this does not require high skills in dev.

**Comment by scoid** [comment]
Yes I would like to get hands on myself, to make anything more translatable or providing it with tooltips. I will have a look into the code. I made pull requests so far with other projects on github.
